README.rst

```diff
@@ -43,6 +44,12 @@ Usage
     schedule.every().wednesday.at("13:15").do(job)
     schedule.every().minute.at(":17").do(job)
 
+    """ Scheduling with argument in function """
+    def function_job(name):
+        print(f"I am {name}")
+        
+    schedule.every(10).seconds.do(job, name="Peter Griffin")
```

Comment:
```suggestion
    schedule.every(10).seconds.do(job, name="Peter")
```

Let's keep the readme clean of cultural references :smile:

Reply:
Sure thing

Reply:
@SijmenHuizenga sorry for this but the function name here need to be changed `schedule.every(10).seconds.do(job, name="Peter")` to `schedule.every(10).seconds.do(function_job, name="Peter")`